lib/teiserver/lobby/libs/lobby_restrictions.ex

```diff
@@ -278,4 +309,19 @@ defmodule Teiserver.Lobby.LobbyRestrictions do
     # Returns true if both critera met
     noob_matches > 0 && anti_noob_matches == 0
   end
+
+  @spec is_rotato_title?(String.t()) :: boolean()
+  def is_rotato_title?(title) do
+    title =
+      title
+      |> String.downcase()
+
+    regex = ~r/\b(rotat)/
+
+    matches =
+      Regex.scan(regex, title)
+      |> Enum.count()
```

Comment:
You can avoid both the downcase and the count with the following:
`Regex.match?(~r/rotat/i, title)`

* The [`i` modifier](https://hexdocs.pm/elixir/Regex.html#module-modifiers) makes the regex case insensitive.
* [Regex.match/2](https://hexdocs.pm/elixir/Regex.html#match?/2) returns what you want, without need for capture group or counting anything.